### PR TITLE
Add Italian language

### PR DESCRIPTION
**Pull request by ZinRicky**
- Add Italian support in README.
- Add Italian translation for both app and Electron.
- Improve translation of some terms, following Spotify’s standards (e.g. “tracks” → “brani” instead of “tracks” → “tracce”).

### CODE REVIEW

**Comment by ZinRicky** [comment]
PS: sorry for the closure of the other PR. I made a mistake and lost every change

File: README.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ It has **no ads** and is **absolutely free** to use.
 
 ### Languages
 - English
+- Italiano
```
Comment (staniel359) [inline README.md]: @ZinRicky This file will be updated after new version is released.
Please remove this line for now.

**Review by staniel359 — CHANGES_REQUESTED** [review]
@ZinRicky Please address these changes.

**Review by staniel359 — APPROVED** [review]

**Comment by staniel359** [comment]
@ZinRicky Great job, thank you.

**Comment by filipporomani** [comment]
Oooooof
I started translasting this 2 days ago, almost 25% done lol
Better luck next time!

**Comment by staniel359** [comment]
@filipporomani That's good too.
You can check out ZinRicky's work and make your changes to it.
Or you both can cooperate.

**Comment by staniel359** [comment]
@ZinRicky @filipporomani By the way, your help's needed [here](https://github.com/staniel359/muffon-website) too.
Does anybody want to translate the website part as well?

**Comment by filipporomani** [comment]
Sure! I'm not able to start that right now, btw can u send me a link to the repo? 
Thanks!

**Comment by staniel359** [comment]
@filipporomani https://github.com/staniel359/muffon-website

**Comment by filipporomani** [comment]
Done! Json file translated

**Comment by staniel359** [comment]
@filipporomani Thank you, wrote you there.

**Comment by staniel359** [comment]
@ZinRicky @filipporomani Maybe somebody wants to translate [this one](https://github.com/staniel359/muffon-api/blob/main/config/locales/en.yml) too?

**Comment by ZinRicky** [comment]
> @ZinRicky @filipporomani Maybe somebody wants to translate [this one](https://github.com/staniel359/muffon-api/blob/main/config/locales/en.yml) too?

Since it seems quite easy, here

[it.zip](https://github.com/staniel359/muffon/files/9591330/it.zip)

**Comment by staniel359** [comment]
@ZinRicky Nice, thank you.

**Comment by filipporomani** [comment]
Sure! When I'll be back home I'll translate it!

Il sab 17 set 2022, 11:01 Aleksey Shpakovsky ***@***.***> ha
scritto:

**Comment by staniel359** [comment]
@filipporomani Sorry, already translated.

**Pull request merged**